kvdb-web/src/lib.rs

```diff
@@ -45,6 +45,13 @@ pub struct Database {
 	indexed_db: SendWrapper<IdbDatabase>,
 }
 
+// TODO: implement when web-based implementation need memory stats
+impl parity_util_mem::MallocSizeOf for Database {
+	fn size_of(&self, _ops: &mut parity_util_mem::MallocSizeOfOps) -> usize {
+		0
```

Comment:
Can we use `malloc_size_of_is_0!` for this?

Reply:
@NikVolf how about this one?

Reply:
Seems addressed from here, replaced with macro

Reply:
https://github.com/paritytech/parity-common/pull/292/commits/06f46be5ae0fa7a1a65b71564b1a68fe1443b0f9#diff-c46adc47c3690979fc6bffac4b869d34L49

Reply:
Indeed it is, apologies! :)